### PR TITLE
DOCS-3602 Dimensioning Private Locations Edits

### PR DESCRIPTION
**Pull request by peculiarism**
- Update private location recommendations
- Add CSS `pointer` troubleshooting information
- Update notification variables

### CODE REVIEW

File: content/en/synthetics/private_locations/dimensioning.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ By defining a test type, maximum number of test runs, and total hardware require
 
 To improve dimensioning, split your test assignments based on test types. For example, you can have some private locations run only API and multistep API tests while other private locations run only browser tests.
 
+### Prerequisites
+
+You will need:
```
Comment (peculiarism) [inline content/en/synthetics/private_locations/dimensioning.md:21]: @m-rousse @troisdiz can I get you both to review this when you have time? 🙏🏻

**Review by troisdiz — COMMENTED** [review]
I think we should not assume our customers are using only `Docker`.
Otherwise LGTM for the PL sizing part

File: content/en/synthetics/api_tests/dns_tests.md [hunk 2]
```diff
@@ -105,7 +105,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (m-rousse) [inline content/en/synthetics/api_tests/dns_tests.md:112]: Mega-nitpick, though I see it is not aligned on other tables further down this file, so feel free to ignore!
```suggestion
    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
    | `{{#is_renotify}}`         | Show when monitor renotifies                                        |
    | `{{^is_renotify}}`         | Show unless monitor renotifies                                      |
    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)                       |
    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)                     |
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 3]
```diff
@@ -34,10 +42,10 @@ Additional requirements vary based on the test type for the private location.
 
 | Test type                                     | CPU/Memory/Disk recommendation    |
 | --------------------------------------------- | --------------------------------- |
-| [API tests][1] and [Multistep API tests][2] | 20mCores/5MiB/1MiB per test run   |
-| [Browser tests][3]                           | 150mCores/1GiB/10MiB per test run |
+| [API tests][1] and [Multistep API tests][2] | 100mCores/200MiB/100MiB per test run   |
+| [Browser tests][3]                           | 800mCores/1GiB/500MiB per test run |
 
-For example, Datadog recommends ~ 1.5 core CPU `(150mCores + (150mCores*10 test runs))`, ~ 10GiB memory `(150MiB + (1GiB*10 test runs))`, and ~ 100MiB disk `(10MiB*10 test runs)` for a private location running only Browser tests with a maximum number of concurrent test runs of `10`.
+For example, Datadog recommends ~ 8 core CPU `(150mCores + (150mCores*10 test runs))`, ~ 10GiB memory `(150MiB + (1GiB*10 test runs))`, and ~ 5GiB disk `(500MiB*10 test runs)` for a private location running only Browser tests with a maximum number of concurrent test runs of `10`.
```
Comment (m-rousse) [inline content/en/synthetics/private_locations/dimensioning.md:48]:
```suggestion
For example, Datadog recommends ~ 8 core CPU `(150mCores + (800mCores*10 test runs))`, ~ 10GiB memory `(150MiB + (1GiB*10 test runs))`, and ~ 5GiB disk `(500MiB*10 test runs)` for a private location running only Browser tests with a maximum number of concurrent test runs of `10`.
```

File: content/en/synthetics/private_locations/dimensioning.md (hunk 1 above)
Comment (alai97) [reply content/en/synthetics/private_locations/dimensioning.md:21]:
```suggestion
To get started with dimensioning your private locations, you need the following:
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 4]
```diff
@@ -16,6 +16,14 @@ By defining a test type, maximum number of test runs, and total hardware require
 
 To improve dimensioning, split your test assignments based on test types. For example, you can have some private locations run only API and multistep API tests while other private locations run only browser tests.
 
+### Prerequisites
+
+You will need:
+
+1. Basic understanding of Docker and container orchestration
+2. The private location configuration file mounted with your orchestrator of choice and accessible to your underlying private location containers
```
Comment (alai97) [inline content/en/synthetics/private_locations/dimensioning.md:24]:
```suggestion
2. The private location configuration file mounted with your orchestrator of choice and is accessible to your underlying private location containers.
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 5]
```diff
@@ -16,6 +16,14 @@ By defining a test type, maximum number of test runs, and total hardware require
...
+3. If you are using browser tests with IP blocking (see [Advanced configuration][5]), `sudo` access may be required
```
Comment (alai97) [inline content/en/synthetics/private_locations/dimensioning.md:25]:
```suggestion
3. If you are using [browser tests with IP blocking][5], `sudo` access may be required.
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 6]
```diff
@@ -62,6 +70,12 @@ When multiple workers are associated with a private location, each worker reques
 
 For example, ten tests are scheduled to run simultaneously on a private location that has two workers running. If Worker 1 is running two tests, Worker 1 can request three additional tests to run. If Worker 2 is not running any tests, Worker 2 can request the five following tests. The remaining two tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
 
+### Troubleshooting
+
+#### Browser tests are taking too long to run
+
+Confirm you are not seeing [out of memory issues][6] with your private location deployments. If you have scaled your container instances following our [guidelines](#Define-your-total-hardware-requirements), please reach out to Datadog Support for assistance.
```
Comment (alai97) [inline content/en/synthetics/private_locations/dimensioning.md:77]:
```suggestion
Confirm you are not seeing [out of memory issues][6] with your private location deployments. If you have scaled your container instances following the [guidelines](#Define-your-total-hardware-requirements) above, reach out to [Datadog Support][7].
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 7]
```diff
@@ -71,3 +85,4 @@ For example, ten tests are scheduled to run simultaneously on a private location
 [3]: /synthetics/browser_tests/?tab=requestoptions
 [4]: /synthetics/cicd_integrations
 [5]: /synthetics/private_locations/configuration#advanced-configuration
+[6]: https://docs.docker.com/config/containers/resource_constraints/
```
Comment (alai97) [inline content/en/synthetics/private_locations/dimensioning.md:88]:
```suggestion
[6]: https://docs.docker.com/config/containers/resource_constraints/
[7]: /help/
```

File: content/en/synthetics/private_locations/dimensioning.md [hunk 8]
```diff
@@ -62,6 +70,12 @@ When multiple workers are associated with a private location, each worker reques
 
 For example, ten tests are scheduled to run simultaneously on a private location that has two workers running. If Worker 1 is running two tests, Worker 1 can request three additional tests to run. If Worker 2 is not running any tests, Worker 2 can request the five following tests. The remaining two tests can be requested by whichever worker has finished running its test first (any worker that has available slots).
 
+### Troubleshooting
```
Comment (alai97) [inline content/en/synthetics/private_locations/dimensioning.md:73]: How do you feel about adding this section into the Troubleshooting page (https://docs-staging.datadoghq.com/beth.glenfield/update-synthetics-docs/synthetics/troubleshooting/)?

File: content/en/synthetics/troubleshooting/_index.md [hunk 9]
```diff
@@ -98,6 +98,10 @@ To fix it, go edit your recording, open the advanced options of the step that is
 
 {{< img src="synthetics/fix_user_locator.mp4" alt="Fixing User Locator error" video="true" width="100%" >}}
 
+#### I am having issues with a CSS pointer property
+
+Automated browsers do not support emulating the CSS `pointer` media feature. Due to this, synthetics browser tests have `pointer: none` for all tests and devices (laptop/tablet/mobile).
```
Comment (alai97) [inline content/en/synthetics/troubleshooting/_index.md:103]:
```suggestion
Automated browsers do not support emulating the CSS `pointer` media feature. Browser tests have `pointer: none` for all tests and devices (laptop, tablet, or mobile).
```

File: content/en/synthetics/api_tests/udp_tests.md [hunk 10]
```diff
@@ -100,7 +100,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/udp_tests.md:107]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/tcp_tests.md [hunk 11]
```diff
@@ -99,7 +99,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/tcp_tests.md:106]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/ssl_tests.md [hunk 12]
```diff
@@ -114,7 +114,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/ssl_tests.md:121]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/icmp_tests.md [hunk 13]
```diff
@@ -100,7 +100,11 @@ Your test sends a notification based on the [alerting conditions](#define-alert-
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/icmp_tests.md:107]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/http_tests.md [hunk 14]
```diff
@@ -177,7 +177,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/http_tests.md:184]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/grpc_tests.md [hunk 15]
```diff
@@ -107,7 +107,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/grpc_tests.md:114]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/api_tests/websocket_tests.md [hunk 16]
```diff
@@ -118,7 +118,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```
Comment (alai97) [inline content/en/synthetics/api_tests/websocket_tests.md:125]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |                           |
```

File: content/en/synthetics/browser_tests/_index.md [hunk 17]
```diff
@@ -132,6 +132,10 @@ A notification is sent according to the set of alerting conditions. To configure
     | `{{^is_alert}}`            | Show unless monitor alerts                                          |
     | `{{#is_recovery}}`         | Show when monitor recovers from either ALERT   |
     | `{{^is_recovery}}`         | Show unless monitor recovers from either ALERT |
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |
```
Comment (alai97) [inline content/en/synthetics/browser_tests/_index.md:138]:
```suggestion
    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/synthetics/private_locations/dimensioning.md (hunk 8 above)
Comment (peculiarism) [reply content/en/synthetics/private_locations/dimensioning.md:73]: Let's do it! I wasn't sure where was appropriate 😄

**Review by alai97 — APPROVED** [review]

**Review by m-rousse — APPROVED** [review]
All good from synthetics wokrer PoV

File: content/en/synthetics/private_locations/dimensioning.md [hunk 18]
```diff
@@ -34,31 +42,31 @@ Additional requirements vary based on the test type for the private location.
...
+ 1. Set the [`concurrency` parameter][4] to `maximum number of test runs that can be executed on your private location / number of workers associated with your private location`.
  2. Assign `total private location resource requirements / number of workers` resources to every private location container.
 
 
-For example, Datadog recommends ~ 1.5 core CPU, ~ 10GiB memory, and ~ 100MiB disk for a private location running only Browser tests with a maximum number of concurrent test runs of `10`. To distribute these resources across two workers, set the [`concurrency` parameter][5] to 5 and allocate ~ 750mCores CPU, ~ 5GiB memory, and ~ 50MiB disk to each worker.
+For example, Datadog recommends ~ 1.5 core CPU, ~ 10GiB memory, and ~ 100MiB disk for a private location running only Browser tests with a maximum number of concurrent test runs of `10`. To distribute these resources across two workers, set the [`concurrency` parameter][4] to 5 and allocate ~ 750mCores CPU, ~ 5GiB memory, and ~ 50MiB disk to each worker.
```
Comment (m-rousse) [inline content/en/synthetics/private_locations/dimensioning.md:65]:
```suggestion
For example, Datadog recommends ~ 8 cores CPU, ~ 10GiB memory, and ~ 5GiB disk for a private location running only Browser tests with a maximum number of concurrent test runs of `10`. To distribute these resources across two workers, set the [`concurrency` parameter][4] to 5 and allocate ~ 4 cores CPU, ~ 5GiB memory, and ~ 2.5GiB disk to each worker.
```

**Pull request merged**